### DISCUSSION TITLE
Hoarder - 自托管 AI 书签管理软件 - 1nan

### DISCUSSION THREAD

**Discussion by giscus** [yinan-c/pages-comment#14] [Announcements]
# Hoarder - 自托管 AI 书签管理软件 - 1nan

https://yinan.me/hoarder-zh.html

**Comment by miantiao-me** [comment]
omnivore 要停止服务了。试了一下 Hoarder，满足我的期望， 已经部署起来

**Reply by yinan-c** [reply]
嗯，Hoarder 已经很可用了，之后肯定会越来越完善

**Comment by versun** [comment]
注意常备份，我之前在升级linkding时，数据搞没了，哭死。之后转到raindrop了。。。

**Reply by yinan-c** [reply]
嗯，我现在还是继续用 Anybox 和 DEVONthink。 之前用过一段时间 raindrop （应该算是我用的第一个书签管理软件），因为是免费版没有离线存档，很多链接已经失效了。

**Reply by yinan-c** [reply]
linkding 也试过一次，看来要避雷了

**Comment by DayuGuo** [comment]
Devonthink 论坛里有一个插件叫GPT Add Tags，也可以试试!

**Reply by yinan-c** [reply]
嗯，感谢推荐！这就去试试。
